teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -152,6 +153,16 @@ public class ValidatorOptions {
   private boolean isLocalSlashingProtectionSynchronizedEnabled =
       DEFAULT_VALIDATOR_IS_LOCAL_SLASHING_PROTECTION_SYNCHRONIZED_ENABLED;
 
+  @Option(
+      names = {"--stop-vc-when-validator-slashed-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description =
+          "If an owned validator key is detected as slashed, the validator-client should terminate. In this case, the service should not be restarted.",
```

Comment:
we need something around this - either exits with code 2 should not be restarted, or something... how do they know how to 'not restart'?

Reply:
I renamed the option to `shut-down-when-validator-slashed-enabled ` to make it clear that this could shut down the VC if run separately or both VC and BN if running in a single process.
I also added a mention of the exit code.
PS: There is a documentation PR with more details about the feature https://github.com/Consensys/doc.teku/pull/526